build/build-extensions.ps1

```diff
@@ -52,7 +52,7 @@ function BuildRuntime([string] $targetRid, [bool] $isSelfContained) {
       $suffixCmd = "/p:VersionSuffix=$suffix"
     }
 
-    $cmd = "publish", ".\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj", "-r", "$targetRid", "--self-contained", "$isSelfContained", "/p:PublishReadyToRun=true", "/p:PublishReadyToRunEmitSymbols=true", "-o", "$publishTarget", "-v", "m", "/p:BuildNumber=$buildNumber", "/p:IsPackable=false", "/p:CommitHash=$commitHash", "-c", "Release", $suffixCmd
+    $cmd = "publish", "$PSScriptRoot\..\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj", "-r", "$targetRid", "--self-contained", "$isSelfContained", "/p:PublishReadyToRun=true", "/p:PublishReadyToRunEmitSymbols=true", "-o", "$publishTarget", "-v", "m", "/p:BuildNumber=$buildNumber", "/p:IsPackable=false", "/p:CommitHash=$commitHash", "-c", "Release", $suffixCmd
```

Comment:
Could you please validate the path before setting it? E.g., 

```powershell
$projectPath = "$PSScriptRoot\..\src\WebJobs.Script.WebHost\WebJobs.Script.WebHost.csproj"

if (-not (Test-Path $projectPath))
{
    throw "Project path '$projectPath' does not exist."
}
```

Reply:
is `$PSScriptRoot` the current dir of the script? I think we get that directory above -- probably in the wrong way b/c I didn't know about that :-) -- with 

```
$currentDir = Split-Path -Parent $MyInvocation.MyCommand.Path
```

Could you remove that line and replace `$currentDir` with `$PSScriptRoot`? It looks like it's only used one time. But that'd make things consistent (and teach others something new!)